napari/_tests/test_interactive_transforms.py

```diff
@@ -0,0 +1,130 @@
+import numpy as np
+import pytest
+from vispy import keys
+
+from napari.components._interaction_box_constants import Box
+from napari.utils.transforms import Affine
+
+
+def check_corners_of_interaction_box(

```

Comment:
I like this change because it makes some of the tests below much more readable - hope it makes sense to you too and wasn't too painful to refactor!

However, the box won't be axis-aligned right? So won't you need to check all 4 corners instead of just the top-left and bottom-right?

Also, I would consider making the `corner` arguments keyword only arguments to make it clear what they are when this is being called.

Reply:
Both good points! In these tests the box is axis-aligned, but I renamed the function to make this clear. Also followed your advice to make keyword-only arguments.